packages/web/src/pages/explore-page/collections.ts

```diff
@@ -51,6 +52,17 @@ export const PREMIUM_TRACKS: ExploreCollection = {
   cardSensitivity: WIDE_CARD_SENSITIVTY
 }
 
+export const DOWNLOADS_AVAILABLE: ExploreCollection = {
+  variant: ExploreCollectionsVariant.DIRECT_LINK,
+  title: 'Downloads Available',
+  subtitle: 'Popular tracks with downloads you can use in your own tracks.',
+  gradient: 'linear-gradient(138deg, #FF00F5 -5.01%, #00D1FF 110.47%)',
+  shadow: 'rgba(9, 175, 233, 0.35)',
+  icon: IconRemix,
+  link: EXPLORE_PREMIUM_TRACKS_PAGE, // TODO make a downloads page
```

Comment:
Heads up this will just link to search tracks with downloads available rather than a new page